packages/directory-fetcher/src/index.ts

```diff
@@ -10,7 +11,14 @@ export interface DirectoryFetcherOptions {
   manifest?: DeferredManifestPromise
 }
 
-export default () => {
+export interface CreateDirectoryFetcherOptions {
+  includeOnlyPackageFiles?: boolean
```

Comment:
Reading the related issue, seems like `includeOnlyPackageFiles` will be false by default. Do you see any scenario where it should be set to true?

Reply:
Yes, I would add such option in the future. It is useful because it allows you to test a more production-like version of your package.